mlrun/model_monitoring/api.py

```diff
@@ -552,7 +556,11 @@ def get_sample_set_statistics(
     :param sample_set:                   A sample dataset to give to compare the inputs in the drift analysis.
     :param model_artifact_feature_stats: The `feature_stats` attribute in the spec of the model artifact, where the
                                          original sample set statistics of the model was used.
-
+    :param sample_set_columns: The column names of sample_set.
+    :param sample_set_drop_columns: ``str`` / ``int`` or a list of ``str`` / ``int`` that
+                                    represent the column names / indices to drop.
```

Comment:
But these will only be dropped if `sample_set` is not already a dataframe? That's not what a user might expect.